content/guides/dogstatsd.md

```diff
@@ -25,315 +25,330 @@ sidebar:
+  # Render the page...
+EOF
 %>
 
-will only be sent to the server about half of the time, but it will be
-multipled by the sample rate to provide an estimate of the real data.
+This produces the same five metrics shown in the Timers section above: count, avg, median, max, and 95percentile.
 
-## Tags
+But histograms aren't just for measuring times. You can track distributions for anything, like the size of files users upload to your site:
 
-Tags are a Datadog specific extension to StatsD. They allow you to tag a metric
-with a dimension that's meaningful to you and slice and dice along that
-dimension in your graphs. For example, if you wanted to measure the
-performance of two video rendering algorithms, you could tag the rendering time
-metric with the version of the algorithm you used.
+<%=python <<EOF
+from datadog import statsd
 
+def handle_file(file, file_size):
+  # Handle the file...
 
-Since tags are an extension to StatsD, so you'll need to use a client that
-supports them.
+  statsd.histogram('mywebsite.user_uploads.file_size', file_size)
+  return
+EOF
+%>
 
+Histograms are an extension to StatsD, so you'll need to use a [DogStatsD client library](/libraries).
 
-<%= python <<eof
+#### Sets
 
-# Randomly choose which rendering function we want to use ...
-if random() < 0.5:
-    renderer = old_slow_renderer
-    version = 'old'
-else:
-    renderer = new_shiny_renderer
-    version = 'new'
+Sets count the number of unique elements in a group. To track the number of unique visitors 
+to your site, use a set:
 
-start_time = time()
-renderer()
-duration = time() - start_time
-dog.histogram('rendering.duration', duration, tags=[version])
-eof
+<%= python <<EOF
+def login(self, user_id):
+    statsd.set('users.uniques', user_id)
+    # Now log the user in ...
+EOF
 %>
 
-## Events
-You can post events to your Datadog event stream. You can tag them, set priority and even aggregate them with other events.
+Sets are an extension to StatsD, so you'll need to use a [DogStatsD client library](/libraries).
 
-Mandatory fields:
+#### Metric option: Sample Rates
 
-  - `title` (String) — Event title.
-  - `text` (String) — Event text. Supports line breaks.
+Since the overhead of sending UDP packets can be too great for some performance
+intensive code paths, DogStatsD clients support sampling,
+i.e. only sending metrics a percentage of the time. The following code sends
+a histogram metric only about half of the time:
 
-Events are aggregated on the Event Stream based on: <br/>
-'hostname/source_type/aggregation_key'<br/>
+<%= python <<EOF
+dog.histogram('my.histogram', 1, sample_rate=0.5)
+EOF
+%>
 
-<%= python <<eof
+Before sending the metric to Datadog, DogStatsD uses the `sample_rate` to 
+correct the metric value, i.e. to estimate what it would have been without sampling.
 
-# Post a simple message
-statsd.event('There might be a storm tomorrow', 'A friend warned me earlier.')
+You can specify a sample rate for any metric type.
```

Comment:
Seems like we should remove the 'sample_rate' option from DogStatsD client libraries.